br/pkg/restore/client.go

```diff
@@ -2403,13 +2461,16 @@ func (rc *Client) InitSchemasReplaceForDDL(
 	var (
 		err    error
 		dbMaps []*backuppb.PitrDBMap
+		// the id map doesn't need to construct only when it is not the first execution
+		idMapNeedConstruct bool
```

Comment:
if `idMapNeedConstruct` is false, we can skip the idmaps reconstruct operation. The operation would downloads all the metakvs.